### PR TITLE
hide company name from right hand rail after first clickthrough

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner @cvializ

### CODE REVIEW

**Comment by andrewconner** [comment]
Works for me

File: kifi-backend/modules/shoebox/marketing [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit bc0b9fbd0f7e6735487fbb7e0b26d4af7efc6ac1
+Subproject commit 437d286b1f5052f446e9e896d6206a51d620f294
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/marketing:1]: :bomb: ?

**Comment by cvializ** [comment]
lgtm

**Comment by camhashemi** [comment]
going to merge asap to beat any updates to the marketing modules, it's up to date atm

**Pull request merged**